### PR TITLE
Use testcontainers for cql tests [full build]

### PR DESCRIPTION
**Pull request by farodin91**
I updated the version of all docker images and worked locally.

I hope that fixes all builds: https://travis-ci.com/farodin91/janusgraph/builds/123602339

The cassandra 2.2 was failed due timeout.
-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by porunov** [comment]
@farodin91 this doesn't fix the issue with `cql` test containers

**Comment by porunov** [comment]
I have a suspicious that test containers are too slow to be tested by Travis CI. Most likely it has too big latency for requests but I am not sure. Please tell if you are OK if I revert the changes

**Comment by farodin91** [comment]
Latency should not be the case. If you calculate diskstorage time together with graphdb time. for murmur to not more than 40 min which is no problem. I think we execute now more tests than before.

**Comment by farodin91** [comment]
I did a small travis run here: https://travis-ci.com/farodin91/janusgraph/builds/123609456

**Comment by farodin91** [comment]
I'm working on SSL for cassandra 3.

**Comment by farodin91** [comment]
I did found a different between old version and new version. 
Before, JanusGraph says it run against a local version of Cassandra which set the number of connection up 1024.
Now JanusGraph says it run against a remote version of Cassandra which set the number of connection up 256. 
I changed this value to 1024.

**Comment by porunov** [comment]
@farodin91 Those CQL tests fails still

**Comment by farodin91** [comment]
Now, just both Cassandra 3 tests are failing, that’s interesting because they run on my machine.

**Comment by porunov** [comment]
@farodin91 Some of those `cql` tests are failing. Also those `cql` tests which passing takes more time to be executed in average. Are you OK if we rollback those commits and merge them when they are fixed?
It is hard to work with the current branches because I don't know if the tests are failed by the changes in the PR or not.

**Comment by porunov** [comment]
@farodin91 I reverted that merge request because it blocks `0.3`, `0.4`, `master` branches. I think it is wise to research and fix the problems with test containers in a separate branch and then merge it when it works

**Comment by farodin91** [comment]
@porunov Do you known how these full build? Because before my changes they didn't work at all for Cassandra and CQL. The version is just set and and nothing is done with it.

**Comment by porunov** [comment]
@farodin91 Actually I don't. @chupman made thise feature to test against all supported Cassandra versions. Isn't it working?

**Comment by farodin91** [comment]
@porunov I fixed a config bug and stopped running all tests again for ssl.

**Comment by farodin91** [comment]
@porunov Should i squash all commits and then merge it.

**Comment by porunov** [comment]
> Should i squash all commits and then merge it.

As you wish. The commits are signed, so they can be committed as separate commits or as a single commit.

The only suggestion would be to enable `[full build]` for merge commits in `0.4` and `master` branches for this PR to see that `[full build]` is consistent with the current `[full build]`.
I would recommend first merge the changes into your local repository (which should be in the same state as JanusGraph repository) and during the merge change the name for merge commits to contain `[full build]`. Then wait in your forked repository till `[full build]` is successful for `0.4` and `master` branches and if everything is OK, we can merge this commit and merge commits.
I.e. the same as I did here: https://github.com/JanusGraph/janusgraph/pull/1730#issuecomment-522318736
Also, I assume your merge commits in `0.4` and `master` will need to include changes from #1735

**Comment by farodin91** [comment]
I have to open a PR for merging it into 0.4. I will squash this branch and merge into 0.3, then i will open a PR to integrate these changes from PR #1735.

**Comment by farodin91** [comment]
The build passed on my travis ci, so i will merge it tomorrow: https://travis-ci.com/farodin91/janusgraph/builds/123678585.

File: TESTING.md [hunk 1]
```diff
@@ -109,6 +109,59 @@ mvn clean install -pl janusgraph-es -Delasticsearch.docker.image=elasticsearch
...
+**Note** Running CQL tests require Docker.
+
+CQL tests are executed using [testcontainers-java](https://www.testcontainers.org/). 
+CQL tests can be executed against a Cassandra 2 using the profile `cassandra2`, 
+a Cassandra 3 using the profile `cassandra3`, or a Scylla 3 using the profile `scylladb`.
+
+```bash
+mvn clean install -pl janusgraph-cql -Pcassandra2-murmur
+mvn clean install -pl janusgraph-cql -Pcassandra3-murmur
+mvn clean install -pl janusgraph-cql -Pscylladb
```
Comment (porunov) [inline TESTING.md]: Is `scylladb` profile tested somehow? Should we add the tests into `.travis.yml` for `[full build]?`

Comment (farodin91) [reply TESTING.md]: I will add it to the issue.

**Comment by mbrukman** [comment]
@jmertic — John, can you please take a look into what happened with the CLA for this PR? It started off as non-compliant, even though you said in another thread the G DATA signed the CLA. Is the CLA being handled properly for G DATA? It seems to have been re-edited by the CLA bot about a dozen times since then, flipping between red and green statuses. Was there an issue of some sort?

Also, since `@communitybridge-easycla` is editing its comment in-place, that means that potential reviewers (who are watching the repo and get all email updates) aren't notified whether this PR is ready for review or not. Is it possible to make it append new comments such that it's clear what the state of the PR is?

**Comment by jmertic** [comment]
@mbrukman Let me ping the EasyCLA product team on this to see if they can figure out what went on.

**Comment by farodin91** [comment]
@mbrukman 
> Is it possible to make it append new comments such that it's clear what the state of the PR is?

I like to new behavior to just get one email.

**Comment by jpalmerLinuxFoundation** [comment]
@jmertic @mbrukman, last week the LF team did a bulk migration of DNS's. Unfortunately, EasyCLA was not migrated properly and we had an outage for multiple hours. This caused false negatives (CLA Not signed) on various projects. I admit I don't totally follow what happened to the commit status. If I have not answered your question, let's connect offline.

In regards to the feature request, I think I'll need to see what the notifications you are getting from GitHub and EasyCLA so I can understand the requirement.

**Pull request merged**